apps/server/src/controllers/userControllers/authUsers.ts

```diff
@@ -288,3 +296,58 @@ export const forgotPassword = async (req: Request, res: Response) => {
     })
   }
 }
+
+export const resetPassword = async (req: Request, res: Response) => {
+  try {
+    const validResetBody = resetPasswordSchema.strict().safeParse(req.body)
+
+    if (!validResetBody.success) {
+      return res.status(HTTP_STATUS_CODE.BAD_REQUEST).json({
+        message: 'Invalid request body.',
+        errors: validResetBody.error,
+      })
+    }
+
+    const { newPassword, email } = validResetBody.data
+    //Let's extract user ID (longstring) from the URL parameter & verify it
+    const { verify } = req.query
+    if (!verify || typeof verify !== 'string') {
```

Comment:
is it possible that if the type exist, it won't be a string?
I think that may be a redundant check

Reply:
ok